tabs/pid_tuning.js

```diff
@@ -701,11 +718,8 @@ TABS.pid_tuning.initRatesPreview = function () {
     this.keepRendering = true;
     this.model = new Model($('.rates_preview'), $('.rates_preview canvas'));
 
-    var scale = d3.scale.linear().domain([900, 2100]);
-
-    this.rollScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.pitchScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.yawScale = scale.range([Math.PI * 2, -Math.PI * 2]);
+    this.$superExpo = $('.pid_tuning input[name="show_superexpo_rates"]');
```

Comment:
I don't quite understand what you mean by 'scope wrapping', but I assume you are referring to how the DOM is manipulated in the project? If so, I agree that this is ultra messy, and the complexity of the DOM is probably beyond what should be done with jQuery. Probably the next big rework of this will have to be to port this to a more appropriate framework.

Adding jQuery elements as object members does not seem to be something that is done elsewhere in the project, or at least not in this tab, so doing it for one element and not for any others adds one more concept to the code, and requires everybody else working on the codebase to be aware and understand this concept. This makes it more likely that other devs will not reimplement stuff without need, which in turn will make it even more messy.